main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1481,4 +1482,65 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
     Name.NName(sp1, idents, sp2)
   }
 
+  /**
+   * Creates the synthetic type classes `Eq` and `Ord`.
+   */
+  private def mkSynthClasses()(implicit flix: Flix): Map[Symbol.ClassSym, ResolvedAst.Class] = {
+    val eqTypeClass = mkEqClass()
+    val ordTypeClass = mkOrdClass(eqTypeClass)
+
+    Map(eqTypeClass.sym -> eqTypeClass, ordTypeClass.sym -> ordTypeClass)
+  }
+
+  /**
+   * Creates the synthetic type class `Eq`.
```

Comment:
This is temporary code, but it would be nice to have some syntax in the comments, so we can quickly see the structure, e.g.:

```
trait Eq[a] {
  def equals(x: a, y: a): Bool
}
```

(we may call `trait` something else, e.g. `class`.)